server/src/ai/ai.router.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import { makeSummary, getCoursesWithMinReviews } from './ai.functions';
+
+const aiRouter = express.Router();
+
+aiRouter.use(express.json());
+
+/** Reachable at POST /api/summarizeReviews
+ * @body a block of text containing all reviews from a course
+ * returns a summary created by OpenAI
+*/
+aiRouter.post('/summarizeReviews', async (req, res) => {
```

Comment:
`/reviews/summarize/`

if we do the api.use('ai', ..) as before then this endpoint is reachable at `api/ai/reviews/summarize`